src/meta/src/rpc/service/hummock_service.rs

```diff
@@ -160,7 +160,7 @@ where
             })),
             Some(mut compact_task) => {
                 self.hummock_manager
-                    .report_compact_task(req.context_id, &mut compact_task)
+                    .report_compact_task(req.context_id, &mut compact_task, None)
```

Comment:
We always pass None for table_stats_change. Is this intentional?

Reply:
It's a bug. Thanks!